### DISCUSSION TITLE
I really want to be able to use the ShortCommands Plugin.

### DISCUSSION THREAD

**Discussion by aniio** [Pryaxis/TShock#2055] [Q&A]
As title says, I really want to use the plugin, to make a command that gives me a ton of buffs, but,
1. it's not updated for 1.4, obviously.
2. I have no clue how to update a plugin, but I know there is something about changing the source code, but then, I don't know how to put the source code back into a dll.

Can someone please help me figure out how to update it, or update it for me?
I would prefer trying to learn to do it, so I can do it on my own later or help others, but if I can't figure it out, then I don't know.

All help is welcome!

(Plugin link: https://tshock.co/xf/index.php?resources/shortcommands.10/)

**Comment by tieonlinux** [comment]
I've an updated version  
Dll is available here: https://github.com/tieonlinux/ShortCommands/releases

Btw if you want to edit plugins and build dlls you should start by using some IDE like rider, visual studio, vs code, ...
